service-front/module/Application/test/Services/SiriusApiServicePactTest.php

```diff
@@ -197,4 +197,55 @@ public function testAbandonCase(): void
         $this->assertEquals(204, $response['status']);
         $this->assertEquals("", $response['error']);
     }
+
+    public function testSendPostOfficePdf(): void
+    {
+        $details = [];
+        $details["firstName"] = "Joe";
+        $details["lastName"] = "Blogs";
+        $details["address"]["line1"] = '123 Ferndale Road';
+        $details["address"]["line2"] = 'Lambeth';
+        $details["address"]["line3"] = 'Line 3';
+        $details["address"]["town"] = 'London';
+        $details["address"]["country"] = 'England';
+        $details["address"]["postcode"] = 'SW4 7SS';
+        $details["lpas"][0] = "789";
```

Comment:
CaseData holds the UID (e.g. M-1234-5678-AB73): you'll need to make a request to the Sirius API to convert this to the internal ID (numeric)

Reply:
Does this matter for this test?